spec/integ/irc-to-matrix.spec.js

```diff
@@ -115,10 +115,23 @@ describe("IRC-to-Matrix message bridging", function() {
         });
     });
 
-    it("should bridge IRC topics as Matrix m.room.topic",
-    function(done) {
+    it("should bridge IRC topics as Matrix m.room.topic in aliased rooms",
+    test.coroutine(function*() {
```

Comment:
This is now a coroutine, meaning the test ends when this function returns. This function returns too early, before `cli.sendStateEvent.andCallFake` fires (and it would be calling a non-existent function `done()`).